web/src/components/molecules/Asset/AssetListTable/hooks.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+
+import { ArchiveExtractionStatus } from "@reearth-cms/components/molecules/Asset/asset.type";
+import { useT } from "@reearth-cms/i18n";
+
+export default (archiveExtractionStatus: ArchiveExtractionStatus) => {
+  const t = useT();
+
+  const [status, setStatus] = useState<string>("");
+  const [statusColor, setStatusColor] = useState<string>("");
+
+  const RED = "#F5222D";
+  const ORANGE = "#FA8C16";
+  const GREEN = "#52C41A";
```

Comment:
Since these aren't being reused I think its fine to just put the color values in directly.
If you prefer them to be assigned to these values, please move the variables outside of the hooks function since they are never going to change.

Reply:
I used the color values directly, Done!